src/renderer/screens/manager/AppsList/AppsList.js

```diff
@@ -154,7 +156,7 @@ const AppsList = ({
         dispatch={dispatch}
         isIncomplete={isIncomplete}
         addAccount={addAccount}
-        disabled={update.length >= 1}
+        disabled={update.length >= 1 || hasAccounts}
```

Comment:
This also has the side effect of not showing the install success banner after I install ETH if I already have BTC accounts for instance, I don't know if this is something we want from product's perspective. To me the story reads as _"don't show the success banner _if coming from a fw update reinstall_"_ which is different. 🤔 

To me it should still show (once per manager session) after a successful install of a supported app.
cc @cthepot-ledger

Reply:
yep you need to create a `hasCurrencyAccounts` of use the `currenciesSelector` to compare the list you get with what you installed :thinking: